pocket_lab/pl_gui_views.h

```diff
@@ -108,9 +108,10 @@ int32_t pl_gui_create_dmm_view(lv_obj_t* parent,
 int32_t pl_gui_create_capture_view(lv_obj_t* parent,
 				   struct pl_gui_init_param *param);
 int32_t pl_gui_create_analysis_view(lv_obj_t* parent,
-			       struct pl_gui_init_param *param);
+				    struct pl_gui_init_param *param);
 int32_t pl_gui_create_about_view(lv_obj_t* parent,
 				 struct pl_gui_init_param *param);
```

Comment:
nit: should this be right indented?

Reply:
This is what I get when I apply astyle even though I manually try to intend to right. I would keep what astyle does